src/GafferImage/OpenColorIOTransform.cpp

```diff
@@ -108,6 +126,43 @@ void OpenColorIOTransform::hashColorData( const Gaffer::Context *context, IECore
 	hashTransform( context, h );
 }
 
+OpenColorIO::ConstContextRcPtr OpenColorIOTransform::getLocalContext(OpenColorIO::ConstConfigRcPtr config) const
```

Comment:
Methods beginning with `get` are always paired with methods beginning with `set` in Gaffer. Let's just call this `localContext()` or `ocioContext()` to avoid confusion.